src/systemd.c

```diff
@@ -281,7 +296,7 @@ int cgroup_create_scope(const char * const scope_name, const char * const slice_
 	sd_bus_message_unref(reply);
 	sd_bus_unref(bus);
 
-	return ret;
+	return cgret;
```

Comment:
`cgret` can be initialized to `ECGFAIL` and set to `cgret = 0`, instead of `ret = 0` in line 288, it would save some lines, it is a coding style comment, please feel free to ignore it.